themes/mlibrary/exhibit-builder/exhibits/summary.php

```diff
@@ -63,7 +63,11 @@ class="twitter-share-button"
                            <?php
                               $page_card_info = mlibrary_display_exhibit_card_info($rawAttachment,$block,$exhibitPage);
                               $uri = exhibit_builder_exhibit_uri($exhibit, $exhibitPage);
-                              echo '<a href="' . html_escape($uri) .'">'.$page_card_info["image"].$page_card_info["title"].$page_card_info["description"].'</a>';
+                              echo '<a href="' . html_escape($uri) .'">';
+                              echo '<div>'.$page_card_info["image"].'</div>';
+                              echo '<div class="card-info"><h3>'.$page_card_info["title"].'</h3>';
+                              echo $page_card_info["description"].'</div>';
+                              echo '</a>';
                            ?>
```

Comment:
Since this is a template file, I'd prefer it if the html in string literals were generated as html, rather than from the php output.  i.e.

```
<a href="<?php print html_escape($uri); ?>">
<div><?php print ... ?></div>
</a>
```